components/library/lasso/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+#
+
+
+include ../../../make-rules/shared-macros.mk
+
+BITS=64
+PATH=$(PATH.gnu)
+BASEURL=	https://dev.entrouvert.org
+
+COMPONENT_NAME=		lasso
+COMPONENT_VERSION=	2.6.0
+COMPONENT_FMRI= library/lasso
+COMPONENT_SUMMARY= Lasso library
+COMPONENT_CLASSIFICATION= System/Libraries
+COMPONENT_SRC=    $(COMPONENT_NAME)-$(COMPONENT_VERSION)
```

Comment:
It seems that some space separators in the `COMPONENT_` block are tabs and some spaces, make it consistent, I prefer tab(s).